### DISCUSSION TITLE
Different way to confirm accepting the EULA

### DISCUSSION THREAD

**Discussion by ddang94** [lawand-dot-io/taskbar#14] [Feature requests]
For permissions in the beginning, you may want to change the eula to a check box to confirm that they have read it before being able to accept the terms. Just in case for legal reasons, clicking the agreement button alone may not be enough to justify it being read.

**Comment by lawand-dot-io** [comment]
Nice suggestion. I'll consider it.

**Comment by lawand-dot-io** [comment]
The request related to dock hiding was moved here: https://github.com/lawand-dot-io/taskbar/discussions/29
